autoload/vital/__vital__/VS/Vim/Window/FloatingWindow.vim

```diff
@@ -189,7 +189,7 @@ endfunction
 " @param {number} args.col 0-based indexing
 " @param {number} args.width
 " @param {number} args.height
-" @param {boolean?} args.border
+" @param {boolean|[string]?} args.border - boolean, or list of characters clockwise from top-left
```

Comment:
Could you please note that it complies with the neovim specifications?

Reply:
I'd be happy to if I knew what specification to look for. :)  I've never seen this particular style of doc comments before - could I get a link?